.agents/skills/dsom-signature-injector/scripts/inject.py

```
@@ -0,0 +1,90 @@
+                    new_content = header + content
+
+                with open(filepath, 'w', encoding='utf-8') as f:
+                    f.write(new_content)
+                print(f"Prepended SH/YML header to {filepath}")
+            elif filepath.endswith('.ps1'):
+                header = get_ps1_header(date_str)
+                new_content = header + content
+                with open(filepath, 'w', encoding='utf-8') as f:
+                    f.write(new_content)
+                print(f"Prepended PS1 header to {filepath}")
+        except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)